consumer-rest-core/src/main/scala/za/co/absa/spline/consumer/rest/model/WriteModeQuery.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.consumer.rest.model
+
+object WriteModeQuery {
+  def parse(str: String): Array[Option[Boolean]] = {
+    str.split(",").map(s => {
```

Comment:
```suggestion
    str.split(",").map(_.trim).map(s => {
```
In rare case, someone sends a space there...

Reply:
That's why one should never save on unit tests :smile: That `trim` used to be there before commit, but somehow vanished at the last moment :smile: :smile: Thank you Adam!

Reply:
Ah, no, my bad, it was in another PR